_pages/en_US/troubleshooting.txt

```diff
@@ -276,13 +276,19 @@ If your console is not on those firmwares, it likely indicates that you already
 <summary><u>"An error has occurred, forcing the software to close..." (white message box)</u></summary>
 
 There is an issue with your `otherapp.bin` file (it is missing, misplaced, or corrupted). Download the latest release of [universal-otherapp](https://github.com/TuxSH/universal-otherapp/releases/latest) and place it on the root of your SD card.
+
+If the above doesn't fix your issue, and you are using an Old 3DS / Old 3DS XL / Old 2DS, you may be encountering a specific issue involving cartridge updates. Take another look at the system version. If the last number, e.g. 4 in 11.3.0-**4**U, is 4 and lower and the system version is 11.3.0 and lower, replace the soundhax file on the root of your SD card with the one from [this site](http://soundhax.saru.moe/frankenfirm.html?crash)
```

Comment:
"specific" is not necessary here
4 and lower -> 4 or lower
soundhax should be capitalized
"If the last number is 4 or lower (e.g. 11.3.0-**4**U) would be preferred

Add link to NH here too? And or maybe to safecerthax?

Reply:
should ``11.3.0 and lower`` also be ``11.3.0 or lower``? (Actually, this part might not even be needed as they shouldn't go soundhax if not?)
not sure about safecerthax, indeed could be an alternative, simply add another section of ``If the above doesn't fix your issue, and you are using an Old 3DS / Old 3DS XL / Old 2DS, you can also try...``?